server/scripts/generate_batch_file.py

```diff
@@ -0,0 +1,83 @@
+import os
+import requests
+import time
+import json
+from tqdm import tqdm
+from dotenv import load_dotenv
+from openai import OpenAI
+
+MODEL='gpt-4o-mini'
+MAX_TOKENS=1000
+TEMPERATURE=0.7
+
+
+script_path = "../data/bird/"
+relative_path='./data/bird/'
+db_change_url="http://localhost:8000/database/change/"
+prompt_generate_url='http://localhost:8000/prompts/generate/'
+prompt_type='dail_sql'
+num_shots=3
+
+directories = [d for d in os.listdir(script_path) if os.path.isdir(os.path.join(script_path, d))]
+directories=directories[:2]
+
+for i in tqdm(directories,desc=f'Processing Directories'):
+
+    response=requests.post(db_change_url,json={'database_type':"hotel","sample_path":f"{relative_path}{i}/sample_questions/{i}_schema.json"})
```

Comment:
Same review, repetitive code. can make a wrapper for create_batch_input_file() and update the file path only.